### DISCUSSION TITLE
Annotate / Higlight in Books in Color?

### DISCUSSION THREAD

**Discussion by Darianna77** [koreader/koreader#12749] [General]
Hello..

I use KoReader on an BigMe Color E-Ink and would love that i could annotate my Books with Color...so this Quote in Red and these Sentence in Blue ;)
Is that possible and i havent found out how? Use it only a week or so so i am a Newbie. I read EPubs

Oh and is it possible that i use my Stylus for the Higlights / markup texts? Have testet and the stylus does nothing in KoReader?

I hope is possible or its a thing that comes in near future?

**Comment by NiLuJe** [comment]
Color highlights are available in current nightlies.

Stylus as something other than a pointer device, not so much ;).

**Comment by Darianna77** [comment]
Oh okay...i think i have the Outside Version. Can i install the nightly over it and my statistics and so on stays? Or how  do i do that correctly. And where can i found the nightly?

Also i ask me what apk i should use....arm or arm64. Dont know that :/ In moment i have install the arm64 i think But dont know if that was right

**Reply by jonnyl2** [reply]
Just switch the Update channel to 'Development' in the Update settings (top hamburger menu). Then do a regular over-the-air update on the device. Settings and statistics will all be kept. 

(If you still need the APK, nightly builds are [here](https://build.koreader.rocks/download/nightly/). If the arm64 was working for you, it was the correct one.)

**Reply by shuvashish76** [reply]
> Can i install the nightly over it and my statistics and so on stays? Or how do i do that correctly. And where can i found the nightly?

See pinned issue thread #11068

> what apk i should use....arm or arm64. Dont know that :/

Most modern devices support arm64, still....install [App Manager](https://f-droid.org/en/packages/io.github.muntashirakon.AppManager/) > Open "Settings" > "About the device" > check "Supported architectures"

**Comment by Darianna77** [comment]
Yeah thy you two. Have switched do Dev and the Color annotate works fine :) No i hope that sometime the BigMe is supportetd so that the lightsettings and the rotation-change works to. But i Love KoReader so far! Thx the Devs for the App :)
